### ISSUE TITLE
[Adaptive Cards - Activity Update] -Ensure links are distinguished from surrounding text in a way that does not rely on color.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8351]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 22H2 (OS Build 25390.1000)
Browser: Edge Dev (111.0.1661.27)
Tool: Accessibility Insight for Web

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Navigate to 'Activity Update' option and press enter key to activate it.
4. Run the accessibility insight for web tool and observe the issue. 

**Actual Result:**
Links are not distinguished from surrounding text in a way that does not rely on color.

**Expected Result:**
Ensure links should distinguished from surrounding text in a way that does not rely on color.

**Element path:**
.w3-col.m6.w3-margin-top > p > a[target="_blank"][rel="noopener"]

**How to fix:**
Fix any of the following:
The link has insufficient color contrast of 2.97:1 with the surrounding text. (Minimum contrast is 3:1, link text: #0050c7, surrounding text: #000000)
The link has no styling (such as underline) to distinguish it from the surrounding text

**Note:**

1. Same issue is repro on the 'Action.Show Card' screen.
2. Same issue is repro under 'Action.ToggleVisibility' screen.

**User Impact:**
Users with colorblindness and low vision have difficulty in identifying the links present on the page. Users can't be able to differentiate between the normal text and links if links do not have underline/chevron symbols.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![image](https://user-images.githubusercontent.com/95913748/223432313-384f4abe-ea29-4fc7-aa00-ef09f31331be.png)

![WCAG1 4 1-Action Show Cards](https://github.com/microsoft/AdaptiveCards/assets/95913748/cd7053ec-e15b-4dbc-965f-d42431b45b2b)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/6c401b87-e6d6-4f81-b90c-f4ca94cf3d4a)

**Comment by anna-dingler** [comment]
Hi @vagpt,

This issue is fixed on our internal site. I expect us to make another website release this month, so I am marking this as completed.

Please let me know if you have any questions!

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as issue is not repro.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/24b0b1be-f83f-4781-83d4-e2454509ab09)
